tests/chaos/oadp/test_oadp.py

```diff
@@ -39,3 +39,31 @@ def test_reboot_vm_node_during_backup(
     oadp_backup_in_progress.wait_for_status(
         status=oadp_backup_in_progress.Backup.Status.PARTIALLYFAILED, timeout=TIMEOUT_10MIN
     )
+
+
+@pytest.mark.destructive
+@pytest.mark.chaos
+@pytest.mark.parametrize(
+    "rhel_vm_with_dv_running",
+    [
+        pytest.param(
+            {
+                "vm_name": "vm-node-cordon-12016",
+                "rhel_image": RHEL_LATEST["image_name"],
+            },
```

Comment:
it seems you have same VM in all tests, the only difference - the vm name. How about to make it class scope and reuse in all tests? in this case you don't need to re-create it all the time

Reply:
Done